index.html

```diff
@@ -625,6 +736,10 @@ <h4 class="modal-title">Help</h4>
             }
             var camera, scene, renderer, light, canvas, controls = null;
 
+            // antialiasing must be disabled per default, so slower devices are not impaired.
+            var antialiasing_mode = false;
+            
+            var stack_dist = 105;
```

Comment:
This change was unavoidable, since the stack distance was determined by the piece size, and the piece size is now variable, which caused the stacks to be badly positioned.